### PR TITLE
updates gulpfile

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

- https://github.com/divio/djangocms-admin-style/issues/250
- adds sourcemaps to gulpfile

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -5,6 +5,7 @@ CHANGELOG
 ------------------
 
 * Fix login password field
+* Enable sourcemaps on gulpfile sass task
```
Comment (FinalAngel) [inline CHANGELOG.rst]: would adapt it like "Enable sourcemaps generation when using ``--debug`` param on Gulp"

File: gulpfile.js [hunk 2]
```diff
@@ -10,6 +10,7 @@
 var autoprefixer = require('gulp-autoprefixer');
 var gulp = require('gulp');
 var gutil = require('gulp-util');
+var gulpif = require('gulp-if');
```
Comment (FinalAngel) [inline gulpfile.js]: you also need to add this to the package.json or?

**Comment by FinalAngel** [comment]
LGTM

File: CHANGELOG.rst [hunk 3]
```diff
@@ -5,6 +5,7 @@ CHANGELOG
 ------------------
 
 * Fix login password field
+* Enable sourcemaps generation when using --debug param on Gulp
```
Comment (vxsx) [inline CHANGELOG.rst]: that's not true

File: gulpfile.js [hunk 4]
```diff
@@ -42,25 +43,19 @@ var PROJECT_PATTERNS = {
 // #TASKS#
 gulp.task('sass', function () {
     gulp.src(PROJECT_PATTERNS.sass)
-        // Sourcemaps are disabled by default to reduce filesize
-        // .pipe(sourcemaps.init())
-        .pipe(sass({ outputStyle: 'compressed' }))
+        .pipe(gulpif(options.debug, sourcemaps.init()))
```
Comment (vxsx) [inline gulpfile.js]: options.debug is undefined

Comment (vxsx) [inline gulpfile.js]: in fact it should from an error, since options are undefined as well

**Comment by vxsx** [comment]
:+1:

**Pull request merged**